src/parser/processor.ts

```diff
@@ -99,7 +99,7 @@ export interface GithubCommentScore {
       content: Record<string, { symbols: { [p: string]: { count: number; multiplier: number } }; score: number }>;
       multiplier: number;
     };
-    relevance?: number;
+    relevance?: number | string;
```

Comment:
I don't think this is a proper solution. The `-` is only a visual change, that should be handled when displayed in a GitHub comment, which is handled by the `github-comment-module`. Changing it to a string or number will introduce lots of type checks and complexifies the use of the relevance value.